<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeSelectorTreeViewWidget</class>
 <widget class="QWidget" name="NodeSelectorTreeViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
border:none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="NodeSelectorWidgetLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>32</number>
   </property>
   <property name="bottomMargin">
    <number>23</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="10">
      <widget class="SearchLineEdit" name="leSearch">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
border-radius:6px;
border-width:1px 1px 1px 1px;
border-style:solid;
border-color:rgba(0,0,0,0.1);
min-height:32px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="bForward">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
    padding: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../Resources_macx.qrc">
         <normaloff>:/images/node_selector/chevron-right-active.png</normaloff>
         <disabledoff>:/images/node_selector/chevron-right.png</disabledoff>:/images/node_selector/chevron-right-active.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="searchingText">
       <property name="styleSheet">
        <string notr="true">*{
font-size:17px;
color:#3D3D3D;
}</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="syncSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="lIcon">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="bBack">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    background-color: transparent;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../Resources_macx.qrc">
         <normaloff>:/images/node_selector/chevron-left-active.png</normaloff>
         <disabledoff>:/images/node_selector/chevron-left.png</disabledoff>:/images/node_selector/chevron-left-active.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="7">
      <spacer name="avatarSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lFolderName">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size:17px;
font-weight: bold;
color:#303233;
}</string>
       </property>
       <property name="text">
        <string notr="true">FolderName</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="lOwnerIcon">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="8" alignment="Qt::AlignLeft">
      <widget class="QWidget" name="searchButtonsWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
  border-radius: 12px;
  font-weight: 600;
  height:24px;
  padding-left:8px;
  padding-right:8px;
  background-color: rgba(0, 0, 0, 0.05);
  color:#616366;
}

QToolButton:checked
{
  background-color: #04101E;
  color:#FAFAFB;
}

QToolButton:hover
{
  background-color: rgba(0, 0, 0, 0.1);
  color:#616366;
}


QToolButton:pressed
{
  background-color: rgba(0, 0, 0, 0.2);
  color:#616366;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="cloudDriveSearch">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Cloud drive</string>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources_macx.qrc">
            <normaloff>:/images/node_selector/search_filter/small_cloud_default.png</normaloff>:/images/node_selector/search_filter/small_cloud_default.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="default_icon" stdset="0">
           <string notr="true">qrc:/images/node_selector/search_filter/small_cloud_default.png</string>
          </property>
          <property name="not_change_text_color" stdset="0">
           <bool>true</bool>
          </property>
          <property name="icon_spacing" stdset="0">
           <UInt>6</UInt>
          </property>
          <property name="hover_selected" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">searchFilterButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="backupsSearch">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Backups</string>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources_macx.qrc">
            <normaloff>:/images/node_selector/search_filter/small_database_default.png</normaloff>:/images/node_selector/search_filter/small_database_default.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="default_icon" stdset="0">
           <string notr="true">qrc:/images/node_selector/search_filter/small_database_default.png</string>
          </property>
          <property name="not_change_text_color" stdset="0">
           <bool>true</bool>
          </property>
          <property name="icon_spacing" stdset="0">
           <UInt>6</UInt>
          </property>
          <property name="hover_selected" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">searchFilterButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="incomingSharesSearch">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Incoming Shares</string>
          </property>
          <property name="icon">
           <iconset resource="../../../Resources_macx.qrc">
            <normaloff>:/images/node_selector/search_filter/small_folder_default.png</normaloff>:/images/node_selector/search_filter/small_folder_default.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="default_icon" stdset="0">
           <string>qrc:/images/node_selector/search_filter/small_folder_default.png</string>
          </property>
          <property name="not_change_text_color" stdset="0">
           <bool>true</bool>
          </property>
          <property name="icon_spacing" stdset="0">
           <UInt>6</UInt>
          </property>
          <property name="hover_selected" stdset="0">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">searchFilterButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>9</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="treeViewPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="NodeSelectorTreeView" name="tMegaFolders">
         <property name="font">
          <font>
           <family>SF Pro Display</family>
           <pointsize>-1</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QTreeView {
    show-decoration-selected: 1;
	selection-background-color: #3393df;
	 font-size: 14px;
	font-weight:400;
 	qproperty-indentation: 18; 
	outline:0;
     font-family:SF Pro Display;
}

QTreeView::item:last
{
	padding-left:6px;
}


QTreeView::item:disabled
{
	background-color:transparent;
}


QTreeView::item:hover {
   background-color:transparent;
}


QTreeView::branch:hover {
background-color:transparent;
}


QTreeView::branch:selected {
background-color: #3393df;
}


QTreeView::item:selected {
    background-color: #3393df;
	color: white;
	font-size: 14px;
}

QTreeView::branch {
padding-top:2px;
}

QTreeView::branch:selected:closed:has-children:adjoins-item {
    border-image: none;
	image:url(://images/node_selector/Icon-Small-Arrow-Left-hover.png) ;
}


QTreeView::branch:selected:open:has-children:adjoins-item {
    border-image: none;
    image: url(://images/node_selector/Icon-Small-Arrow-Down-hover.png) ;
}


QTreeView::branch:!selected:closed:has-children:adjoins-item {
    border-image: none;
    image: url(://images/node_selector/Icon-Small-Arrow-Left.png) ;
}


QTreeView::branch:!selected:open:has-children:adjoins-item {
    border-image: none;
    image: url(://images/node_selector/Icon-Small-Arrow-Down.png) ;
}


QHeaderView
{
background: #FFFFFF;
font-size: 12px;
font-weight:400;
color:#303233;
padding-top: 2px;
font-family:SF Pro Display;
}

QHeaderView::section {
background: #FFFFFF;
border-left:1px solid #dadada;			
border-top:0 px;
border-bottom:0px;					 					 				 					 	
border-right:0px;
padding-left:8px;
}


QHeaderView::section::first
{
border-left:0px;		
}

QHeaderView::down-arrow { 
width: 4px; height:6px;
subcontrol-position: right;
top:-2px;
left:0px;
padding-right:10px;
} 

QHeaderView::up-arrow {
width: 4px; height:6px;
subcontrol-position: right;
top:-2px;
left:0px;
padding-right:10px;
} 




</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoExpandDelay">
          <number>-1</number>
         </property>
         <property name="indentation">
          <number>18</number>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerMinimumSectionSize">
          <number>35</number>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>35</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="emptyPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="emptyIcon">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../Resources_macx.qrc">
              <normaloff>:/images/node_selector/view/cloud.png</normaloff>:/images/node_selector/view/cloud.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="searchEmptyInfoWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">*{
color:#3D3D3D;
font-size:16px;
font-weight: 700;
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item alignment="Qt::AlignRight">
              <widget class="QLabel" name="textLabel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>No results for:</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="searchNotFoundText">
               <property name="styleSheet">
                <string notr="true">*{
font-weight: 400;
}</string>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbAlwaysUploadToLocation">
       <property name="text">
        <string>Always upload to this destination</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="bLayout" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
min-height:32px;
padding-left:12px;
padding-right:12px;
font-size:14px;
font-weight: 400;
border-radius:4px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="bNewFolder">
        <property name="styleSheet">
         <string notr="true">
QPushButton
{
background-color:qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 #AAAAAA, stop:1 #999999);
color:#FFFFFF;
border: 1px solid rgba(0, 0, 0, 0.44);
}

QPushButton:disabled
{
background-color: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 rgba(170, 170, 170, 0.3), stop:1 rgba(153, 153, 153, 0.3));
border: 1px solid rgba(0, 0, 0, 0.15);
}

QPushButton::hover
{
background-color:qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 #8D8D8D, stop:1 #7A7A7A);
}</string>
        </property>
        <property name="text">
         <string>New folder</string>
        </property>
        <property name="icon">
         <iconset resource="../../../Resources_win.qrc">
          <normaloff>:/images/node_selector/plus.png</normaloff>:/images/node_selector/plus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background-color:qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 #FFFFFF, stop:1 #FAFAFA);
color:#333333;
border: 1px solid rgba(0, 0, 0, 0.1);
}

QPushButton:disabled
{
color: rgba(51, 51, 51, 0.3);
}

QPushButton:hover
{
background-color:qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 #FAFAFA, stop:1 #F2F2F2);
color:#333333;
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bOk">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
background:#00BFA5;
border: 1px solid rgba(0, 0, 0, 0.1);
color:#FFFFFF;
}

QPushButton:disabled
{
background: rgba(0, 191, 165, 0.3);
color:#FFFFFF;
}

QPushButton:hover
{
background: #00A787;
color:#FFFFFF;
}</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NodeSelectorTreeView</class>
   <extends>QTreeView</extends>
   <header>gui/node_selector/gui/NodeSelectorTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QFrame</extends>
   <header>gui/node_selector/gui/SearchLineEdit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources_macx.qrc"/>
  <include location="../../../Resources_macx.qrc"/>
  <include location="../../../Resources_macx.qrc"/>
  <include location="../../../Resources_macx.qrc"/>
  <include location="../../../Resources_macx.qrc"/>
  <include location="../../../Resources_macx.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="searchFilterButtonGroup"/>
 </buttongroups>
</ui>
